File: stress/t/flt_explog.c
#include <stdio.h>
#include <math.h>
int main(void){
  double x,e,l,l2,l10;
  x=2.5; e=exp(x); l=log(x);
  printf("e= %f\n",e);
  printf("l= %f\n",l);
  x=8.0; l2=log2(x);
  printf("l2= %f\n",l2);
  x=1000.0; l10=log10(x);
  printf("l10= %f\n",l10);
  return 0;
}

File: stress/t/flt_explog.f
      PROGRAM FEXPLOG
      IMPLICIT DOUBLE PRECISION (A-H,O-Z)
      X=2.5D0
      E=EXP(X)
      AL=LOG(X)
      WRITE(*,'(A,F0.6)') 'e= ',E
      WRITE(*,'(A,F0.6)') 'l= ',AL
      X=8.0D0
      AL2=LOG(X)/LOG(2.0D0)
      WRITE(*,'(A,F0.6)') 'l2= ',AL2
      X=1000.0D0
      AL10=LOG10(X)
      WRITE(*,'(A,F0.6)') 'l10= ',AL10
      END
